marvel-app/coordinator/MainCoordinator.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  MainCoordinator.swift
+//  marvel-app
+//
+//  Created by Michal on 11/08/2022.
+//
+
+import Foundation
+import UIKit
+
+//class MainCoordinator: Coordinator {
```

Comment:
Delete comments

Reply:
This file shouldn't be here, because I deleted it